### DISCUSSION TITLE
Consider project export as DAWproject

### DISCUSSION THREAD

**Discussion by ForeverNooob** [apiel/zicBox#4] [Ideas]
[DAWproject](https://github.com/bitwig/dawproject) is an open format for projects which are meant to be cross-compatible between DAWs. Currenly it's supported by Bitwig, Studio One and Reaper (through an external converter tool)

Would be nice to have an open source device that exports in an open format like that IMO.

**Comment by apiel** [comment]
Woooow, this is an amazing hint. I will definitly have a look at this and I am exactly right now working on data serialization, so it might be good timing to switch to a standadize format ;-)
The only annoying part about it, is that I will have to deal with xml file :-/

**Comment by apiel** [comment]
I was looking more in details about DAWproject and it look great. But if I would want to adopt this format, I would have to rethink the whole ZicBox project. Also, at the first glance, it seem that this format is more focus on time line application than for live performance application. Just as an example, if you look at the [`Notes`](https://htmlpreview.github.io/?https://github.com/bitwig/dawproject/blob/main/Reference.html#Note) element, it's limited to only the basic of a midi note. There is no way to assign basic thing like probability. Maybe there would be way to do it with the `Automation Elements`...? As a single developer on this project (at least for the moment), I don't think I have the capability to do such thing.

If I would start the project from scratch, I would for sure consider using this format. But doing it now would be a huge amount of work and I am not even sure that it would fit with this kind of project.
